src/amo/pages/Addon/index.js

```diff
@@ -299,6 +299,34 @@ export class AddonBase extends React.Component {
     ) : null;
   }
 
+  renderDevCommentsCard = () => {
+    const { addon } = this.props;
+    const commentsProps = {};
+    let showComments = true;
+    let comments;
+
+    if (addon && addon.developer_comments) {
+      comments = addon.developer_comments;
+    }
+
+    if (!comments || comments === null) {
+      showComments = false;
```

Comment:
I think we could omit the `showComments` variable if we use a early return statement:

```js
if (!comments) {
  return null;
}
```